css/components/modal/login_modal.scss

```diff
@@ -1,14 +1,52 @@
 .c-login-modal {
```

Comment:
Rename this file to `styles.scss`

Reply:
<img width="246" alt="Screenshot 2019-04-08 at 11 20 41" src="https://user-images.githubusercontent.com/33252015/55713176-67341380-59f0-11e9-9886-275de620479e.png">